test/components/FileInput.spec.js

```diff
@@ -123,7 +125,7 @@ describe('components', () => {
       const { DropArea, fileInput } = setup({ onDropCallback });
 
       const file = { name: 'MockFile.mkv', size: 5000 };
-      fileInput.setState({ isOver: 2, enteredInDocument: 2 });
+      fileInput.setState({ isOver: 2, enteredInDocument: 2, image: '' });
```

Comment:
Shouldn't there be `image: null`?

Reply:
Actually, `setState` should not set any value in that line for `image`.